.github/workflows/run_tests.yml

```
@@ -26,8 +29,16 @@ jobs:
           key: poetry-${{ hashFiles('**/poetry.lock') }}
           restore-keys: |
             poetry-${{ hashFiles('**/poetry.lock') }}
+
       - name: Install dependencies
         run: poetry install
-      - name: Test with pytest
+
+      - name: Test with coverage
         run: |
-          poetry run pytest
+          PYTHONPATH=. poetry run pytest -vvv --cov=common --cov-report term --cov-report=xml
```

Can you please try to run it without `PYTHONPATH`? I think test will work without it.
```suggestion
          poetry run pytest -vvv --cov=common --cov-report term --cov-report=xml
```